packages/svelteui-composables/src/utilities/index.ts

```diff
@@ -1,6 +1,7 @@
 export * from './use-browser-context/index.js';
 export * from './use-debounce/index.js';
 export * from './use-eye-dropper/index.js';
+export * from './use-focus-return/index';
```

Comment:
you need to put `.js` or else it will create problems when trying to import the packaged version